### PR TITLE
Resolves #723 by completing convertTablespoonsToTeaspons method and adding randomInRange skeleton & tests

### LINKED ISSUE #723: Fix convertTablespoonsToTeaspoons method
There is currently a method called convertInchesToYards that lives in /lib/convertTablespoonsToTeaspoons.js.
It is incomplete and needs to be fixed!

This method expects a number as input (tablespoons) and should return a number (teaspoons). You need to make sure it throws an error when the input is not a number, or when the number is negative.

Acceptance Criteria:

Running yarn test convertTablespoonsToTeaspoons results in tests passing.
You have written a skeleton method for someone else to work on.
You have written tests surrounding your skeleton method.
Running yarn lint does not print any errors to the console!
Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.
Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the Contributing Guide.

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by emilyanndavis**
Closes #723 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/convertTablespoonsToTeaspoons.js [hunk 1]
```diff
@@ -4,8 +4,16 @@
...
 
+const teaspoonsInATablespoon = 3;
+
 function convertTablespoonsToTeaspoons(tablespoons) {
-  throw new Error("FIXME");
+  if (typeof tablespoons !== "number") {
+    throw new Error("Tablespoons must be a number.");
+  }
+  if (tablespoons < 0) {
+    throw new Error("Number of tablespoons cannot be negative. Flip your measuring spoons right-side-up and try again.");
```
Comment (BlakeGuilloud) [inline lib/convertTablespoonsToTeaspoons.js]: 😂

**Comment by emilyanndavis** [comment]
New issue created. Thanks!
